Makefile

```diff
@@ -112,3 +113,8 @@ oc-stop-forwarding-ports:
 oc-serve: oc-forward-ports
 	export FLASK_APP=masu; export FLASK_ENV=development; flask run
 	make oc-stop-forwarding-ports
+
+oc-run-migrations: oc-forward-ports
+	sleep 1
+	$(TOPDIR)/tests/create_db.sh
```

Comment:
Can we add this script to run-migrations so it can be generated when running locally?